### PR TITLE
Add FXIOS-9137 ⁃ Modify Edit Bookmark popup menu

### PR DESCRIPTION
**Pull request by dicarobinho**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-9137)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/20245)

## :bulb: Description
Added Delete Bookmark button for Edit Bookmark Screen

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

**Comment by dicarobinho** [comment]
Simulator Screenshot - iPhone 15 Pro Max - 2024-10-01 at 12 09 49

**Comment by adudenamedruby** [comment]
will review this once the delete action has been added as well.

**Comment by dicarobinho** [comment]
https://github.com/user-attachments/assets/be842970-14bb-4808-8a19-6d3b07f3cc9a

File: firefox-ios/Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -172,6 +172,11 @@ extension String {
                 tableName: "Bookmarks",
                 value: "Bookmarks",
                 comment: "Name of the top level bookmarks folder present in the folder selection menu of the`Edit Bookmark` screen")
+            public static let DeleteBookmark = MZLocalizedString(
+                key: "Bookmarks.Menu.DeleteBookmark.v132",
+                tableName: "Bookmarks",
+                value: "Delete Bookmark",
+                comment: "Title for the Delete Bookmark button in the Edit Bookmark screen.")
```
Comment (adudenamedruby) [inline firefox-ios/Client/Frontend/Strings.swift:179]:
```suggestion
                comment: "The title for the Delete Bookmark button, in the Edit Bookmark popup screen which is summoned from the main menu's Save submenu, which will delete the currently bookmarked site from the user's bookmarks.")
```
nit
I always go ham on comments so I avoid getting PRed on them. lolololol

**Review by adudenamedruby — APPROVED** [review]

**Pull request merged**